### ISSUE TITLE
Redefinition of struct timespec in usb package prevents installation on Windows 10

### ISSUE THREAD

**Issue by SkyLined** [arvydas/blinkstick-node#24]
There is an issue with libusb and Windows 10 SDK caused by a redefinition of "struct timespec":

https://github.com/libusb/libusb/issues/144

BlinkStick depends on usb, which is the node package for libusb. The later will fail to build which prevents installation of BlinkStick. I've added a patch and a temporary work-around to the upstream issue: if you're experiencing this issue as well, you may want to check there for a solution.

**Comment by Good4m** [comment]
See the fix I found here:
https://github.com/arvydas/blinkstick-node/issues/26

**Comment by Good4m** [comment]
I found the following fork fixes the issue:
https://github.com/aktary/node-usb

I am using Electron so I also had to run the following command:
$ electron-rebuild -w usb -p

**Comment by mcuee** [comment]
I do not think this is a libusb issue, please updated to latest version of Windows SDK or Visual Studio and try agian.

**Comment by Good4m** [comment]
Turns out it is. Arvydas released a new branch called node-hid which fixes all issues though. If you look on libusb's github there are many conversations about the issue.

**Comment by mcuee** [comment]
Just take note that I am the co-admin of libusb project. I have reopened libusb ticket 144 but I am not convinced it is a libusb issue. 
  https://github.com/libusb/libusb/issues/144
